pytest.ini

```diff
@@ -6,3 +6,4 @@ markers =
     tags: marks tests as tags
     bs: marks tests as business services
     stakeholder: marks tests as stakeholder
+    stakeholder_groups: marks tests as stakeholder stakeholder_groups
```

Comment:
@kpunwatk Please change this:
"stakeholder_groups: marks tests as stakeholder groups"

Reply:
Done. Thanks for noticing